packages/web/src/js/TextAreaGrow.ts

```diff
@@ -0,0 +1,69 @@
+import BaseComponent from './BaseComponent';
+import { enableToggleAutoloader } from './utils';
+import SelectorEngine from './dom/SelectorEngine';
+import EventHandler from './dom/EventHandler';
+
+const NAME = 'textAreaGrow';
+const DATA_KEY = 'textAreaGrow';
+const DATA_TEXTAREA = `data-${DATA_KEY}`;
+const RESIZE_EVENT = 'resize';
+const DEBOUNCE_DELAY = 50;
+
+function debounce<T extends (...args: never[]) => unknown>(func: T, wait: number): (...args: Parameters<T>) => void {
+  let timeout: ReturnType<typeof setTimeout> | null;
+
+  return function debouncedFunction(...args: Parameters<T>) {
+    if (timeout !== null) {
+      clearTimeout(timeout);
+    }
+    timeout = setTimeout(() => func.apply(this, args), wait);
+  };
+}
+
+class TextAreaGrow extends BaseComponent {
+  static get NAME() {
+    return NAME;
+  }
+
+  static get DATA_KEY() {
+    return `${this.NAME}`;
+  }
+
+  constructor(element: HTMLElement) {
+    super(element);
+    this.adjustHeight = debounce(this.adjustHeight.bind(this), DEBOUNCE_DELAY);
+    this.adjustHeight();
+    this.initEvents();
+  }
+
+  initEvents() {
```

Comment:
```suggestion
  addEventListeners() {
```

Please check out other files for consistent naming.

Reply:
ok, but I used collapse as a reference, so it should be fixed there too. But that's another PR.